scalahost/nsc/src/test/scala/scala/meta/tests/scalahost/mirrors/SemanticSuite.scala

```diff
@@ -235,4 +235,14 @@ class SemanticSuite extends DatabaseSuite {
       |[294..294) [C[Int]]
   """.trim.stripMargin
   )
+  messages(
+    """
+      |import scala.collection.mutable.{ Map, Set, ListBuffer }
+      |object a { ListBuffer.empty[Int] }
+    """.stripMargin.trim,
+    """
+      |[0..34..56): [warning] Unused import
```

Comment:
Given the amount of hacky changes necessary to accommodate this use case, I suggest we instead hack at the source - by hijacking the positions of `unused import` warnings.